rolling-shutter/keyper/keypermetrics/metrics.go

```diff
@@ -79,7 +87,17 @@ var MetricsKeyperBatchConfigInfo = prometheus.NewGaugeVec(
 	},
 	[]string{"batch_config_index", "keyper_addresses"})
 
-func InitMetrics() {
+var MetricsKeyperSuccessfulDKG = prometheus.NewGaugeVec(
+	prometheus.GaugeOpts{
+		Namespace: "shutter",
+		Subsystem: "keyper",
+		Name:      "successful_dkg",
```

Comment:
Minor nitpick: I'd suggest to use `dkg_status` instead, that would allow to support multiple different statuses instead of just true/false